src/browser/modules/Sync/BrowserSync.jsx

```diff
@@ -264,6 +264,12 @@ export class BrowserSync extends Component {
 
     return (
       <Drawer id='sync-drawer'>
+        <button
+          onClick={() =>
+            this.props.sendActionToDispatch({ type: 'grass/SYNC_GRASS' })}
+        >
+          Sync grass
+        </button>
```

Comment:
Why do we have this button here? Sync happens automatically, right?

Reply:
👍